tsconfig.json

```diff
@@ -26,6 +26,9 @@
       "@blackbaud/skyux-builder/*": [
         "./node_modules/@blackbaud/skyux-builder/*",
         "./*"
+      ],
+      ".skypageslocales/*": [
```

Comment:
Allow libraries to reference `.skypageslocales` during `skyux serve` and `skyux build`.